test/AngularSpec.js

```diff
@@ -240,6 +240,20 @@ describe('angular', function() {
       }
     });
 
+    it("should handle Uint8Array subarray", function() {
```

Comment:
@gkalpak actually it was already tested by existing cases like `should copy a Int16Array with no destination`. And this is the reason I changed byteLength to length in the initial PR.
Changed the new test anyway, it will be more obvious indeed.